### DISCUSSION TITLE
PCL“游戏垃圾清理”是清理什么路径的文件

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#3079] [提问]
日志我能理解，但是错误报告这件事吧......我都是单独存一个文件夹的啊
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/06db4176-45bc-4fa6-8173-84ad14408e0e)

**Comment by MoYuan-CN** [comment]
错误报告默认是往版本根目录下存的好像..

**Comment by iPodtouch6557** [comment]
顺便就借楼问一下，这个功能加了为啥啊，我个人从来没用过半次:)

**Reply by WorldHim** [reply]
#491

**Comment by Ad-closeNN** [comment]
微微问一下，这个pcl(楼主的)是什么版本，看着挺远古的
我pcl的长这样
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/a031bbcd-d55a-4d4c-8f80-b3721fd06e40)

**Reply by MoYuan-CN** [reply]
PCL 版本？

**Reply by iPodtouch6557** [reply]
我由于瞎存错误报告、游戏日志和缓存，导致这个功能我完全用不上（）

**Reply by kong-que-dnf123** [reply]
更多的“关于与鸣谢”不是会写PCL版本吗🤔

**Reply by Ad-closeNN** [reply]
楼主的pcl版本
~~话说github有楼主这一说吗~~

**Reply by kong-que-dnf123** [reply]
> 楼主的pcl版本 ~话说github有楼主这一说吗~

~要是说这个discussion的发起者的话那不就是我吗~
我PCL一直都是保持更新到最新来着.......

-------------------------------------------------------------------------------------------------------------------------------------------------
EDIT：试了一下，手动更改缓存位置会有这样的提示（但不是红色窗口.jpg）
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/98f95354-8ba4-492c-8297-3c72b99d96bd)

**Reply by Ad-closeNN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/131e1d5a-6399-45b8-99ab-9a0bb3da104a)
"虽然没人......"在新版本被改了？
最新版![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/b03bae43-90eb-4a62-9027-83b9a4ed8b50)
